src/neo/Network/P2P/Payloads/Header.cs

```diff
@@ -35,5 +36,20 @@ public override void Serialize(BinaryWriter writer)
             base.Serialize(writer);
             writer.Write((byte)0);
         }
+
+        public TrimmedBlock Trim()
```

Comment:
Maybe we should rename `TrimmedBlock` to something really related to headers.

Reply:
Could you share some thoughts about the name?

Reply:
I think we don't need this method since we only cache the headers in memory now.

Reply:
This is needed in `LedgerContract`, where Header and Block share the same table.

Reply:
But we don't need to save headers. Now we cache them in memory only.

Reply:
fixed